### PR TITLE
daily test: Switch to support ecs

### PR DESCRIPTION
**Pull request by legendlee1314**
For server.py: just run chromium test only.
For test-x64.py: automatically recognize devices.

### CODE REVIEW

File: python/server.py [hunk 1]
```diff
@@ -116,7 +132,11 @@ def update_webbench():
 
 
 def test_x64_all():
-    return 'test-x64.py --target-arch x86_64,x86'
+    return 'test-x64.py --target-arch x86_64,x86 --device-id ' + device_id
+
+
+def test_x64_chromium():
+    return 'test-x64.py --target-arch x86_64,x86 --phase chromium-x64 --device-id ' + device_id
```
Comment (gyagp) [inline python/server.py]: Please move test_x64_all, test_x64_aosp_build and test_x64_chromium together.

**Comment by gyagp** [comment]
Do not add option --device-id, instead, use our function to probe the devices automatically, and select among them.

**Comment by legendlee1314** [comment]
@gyagp , plz review this patch.

File: python/test-x64.py [hunk 2]
```diff
@@ -12,6 +12,8 @@
 dir_aosp = ''
 dir_chromium = ''
 
+devices_info = {}
```
Comment (gyagp) [inline python/test-x64.py]: devices_info doesn't have to be global variable.

File: python/test-x64.py [hunk 3]
```diff
@@ -80,6 +82,15 @@ def setup():
     else:
         phases = args.phase.split(',')
 
+    (devices_id, devices_product, devices_type, devices_arch, devices_mode) = setup_device()
+
+    for index, device_id in enumerate(devices_id):
+        if devices_arch[index] in target_archs:
+            devices_info[devices_arch[index]] = device_id
```
Comment (gyagp) [inline python/test-x64.py]: Pick the first available one, instead of the last.

File: python/test-x64.py [hunk 4]
```diff
@@ -80,6 +82,15 @@ def setup():
     else:
         phases = args.phase.split(',')
 
+    (devices_id, devices_product, devices_type, devices_arch, devices_mode) = setup_device()
+
+    for index, device_id in enumerate(devices_id):
+        if devices_arch[index] in target_archs and not devices_arch[index] in devices_info:
```
Comment (legendlee1314) [inline python/test-x64.py]: @gyagp , done

**Pull request merged**